app/src/main/java/org/xbmc/kore/host/HostConnectionObserver.java

```diff
@@ -477,6 +577,37 @@ public void onError(int errorCode, String description) {
         }, checkerHandler);
     }
 
+    private boolean isCheckingWhatsPlaying = false;
```

Comment:
Isn't `isCheckingPlaylist` a better name?

Reply:
Indeed it is.